spec/features/admin/admin_updates_user_info_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+feature 'admin updates a user', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.